core/src/main/java/org/apache/carbondata/core/carbon/datastore/DataRefNode.java

```diff
@@ -74,7 +74,7 @@
    * @param blockIndexes indexes of the blocks need to be read
```

Comment:
can you add more description of blockIndexes, it is two dimensional array, what does each dimension mean